src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -114,9 +115,9 @@ const ChatItemTargetText = ({
         )}
         {attachments.length > 0 && <ChatItemAttachment attachments={attachments} />}
         {attachments.length <= 0 && (
-          <Text testID="chat-item-message" style={styles.text}>
-            {message}
-          </Text>
+          <View testID="chat-item-message">
+            <LinkableText style={styles.text} text={message} />
+          </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of `<Text>` with `<View>` and `<LinkableText>` seems correct as per the PR title and description. It would make the text within the chat item clickable if it contains a link. However, please ensure that the `LinkableText` component has been thoroughly tested in various scenarios to avoid any unexpected behavior or crashes.
```diff
- <Text testID="chat-item-message" style={styles.text}>
-   {message}
- </Text>
+ <View testID="chat-item-message">
+   <LinkableText style={styles.text} text={message} />
+ </View>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->